src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -181,3 +183,8 @@ interface TriggeredInfo {
     val user: User?
     val build: Build?
 }
+
+interface BuildDependencyTrigger {
+    val dependsOnBuildConfiguration: BuildConfigurationId
+    val properties: List<Parameter>
```

Comment:
Firstly, why something which returns list of parameters is called 'properties'? Secondly, what parameters a build trigger may have? If I look at TeamCity UI for 'Finish Build Trigger' the only configurable thing is "Trigger after successful build only" option. Why not expose it as explicit boolean property instead of generic list of parameters?